build.gradle

```diff
@@ -107,41 +107,60 @@ ext.libraries = [
 
 group = 'com.linkedin.venice'
 
+publishing {
+  repositories {
+    mavenLocal()
+    maven {
+      name "LinkedInJFrog"
```

Comment:
Nitpick: let's use single-quoted strings for simple text to keep style consistent. Double-quoted strings are used for token expansion (e.g. "${variable}")